app/src/main/java/org/mozilla/fenix/customtabs/CustomTabsIntegration.kt

```diff
@@ -49,8 +49,23 @@ class CustomTabsIntegration(
                 ThemeManager.resolveAttribute(R.attr.shieldLottieFile, activity)
             )
         task.addListener { result ->
-            val lottieDrawable = LottieDrawable()
-            lottieDrawable.composition = result
+            val coloredShield = LottieDrawable().apply {
+                composition = result
+            }
+            val normalShield = AppCompatResources.getDrawable(
+                activity,
+                R.drawable.ic_tracking_protection_enabled
+            )!!
+
+            toolbar.display.icons = toolbar.display.icons.copy(
+                // Custom private tab backgrounds have bad contrast against the colored shield
+                trackingProtectionTrackersBlocked = if (isPrivate) normalShield else coloredShield,
```

Comment:
I believe that we need to use the normal shield on all custom tabs, not just private ones. https://github.com/mozilla-mobile/fenix/issues/5334#issuecomment-562273903

Reply:
🤦‍♂ I can't read.  Thank you @NotWoods!